examples/regional-dlp/main.tf

```diff
@@ -29,9 +29,9 @@ module "data_ingestion" {
   sdx_project_number               = var.sdx_project_number
   terraform_service_account        = var.terraform_service_account
   access_context_manager_policy_id = var.access_context_manager_policy_id
-  bucket_name                      = "dlp-flex-data-ingestion"
-  dataset_id                       = "dlp_flex_data_ingestion"
-  cmek_keyring_name                = "dlp_flex_data-ingestion"
+  bucket_name                      = "dlp-flex-data-ing"
```

Comment:
`dlp` and `flex` don't really mean much when looking at resources within a project.

maybe donate each example with `blueprint-<exampleGoal>`  such as `blueprint-ingestion` and `blueprint-standalone` and `blueprint-default`

Reply:
Mainly looking for consistency in naming no matter which example. 

Looks like you want to match example name in resource name and have all 3 variables be named the same. That is fine. To be consistent in *standalone* plz name all three variables the same rather then name the key *xyz-cmek*